src/makeCalculator.js

```diff
@@ -4,7 +4,23 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate: function (sv, x) {
+      this.result = sv(this.result, x);
+
+      return this;
```

Comment:
You can use method shorthand syntax here instead of the `function` keyword. Replace `operate: function (sv, x)` with `operate(sv, x)`.